opentelemetry-sdk/src/metrics/pipeline.rs

```diff
@@ -414,15 +413,13 @@ where
                 if existing == id {
                     return;
                 }
-
-                global::handle_error(MetricsError::Other(format!(
-                    "duplicate metric stream definitions, names: ({} and {}), descriptions: ({} and {}), kinds: ({:?} and {:?}), units: ({:?} and {:?}), and numbers: ({} and {})",
-                    existing.name, id.name,
-                    existing.description, id.description,
-                    existing.kind, id.kind,
-                    existing.unit, id.unit,
-                    existing.number, id.number,
-               )))
+                otel_warn!(name: "InstrumentSync.AddSync.DuplicateMetricStreamDefinitions",
```

Comment:
These names must be user friendly, and not internal implementation name based.

Reply:
Have updated the names, and also reverted changes from periodic reader.